docs/guide/docs/channels/messenger.md

```diff
@@ -5,82 +5,139 @@ title: Facebook Messenger
+  - Use Serveo to create an HTTPS tunnel to your machine. [**Tutorial**](https://medium.com/automationmaster/how-to-forward-my-local-port-to-public-using-serveo-4979f352a3bf)  
 
 ## Setup
 
-### Global Configuration
+### Enable the Messenger Channel
 
-#### Enable the Messenger Channel
+You can do this by opening the file `data/global/botpress.config.json` in a text editor and setting the value to `true` as below. 
 
-- Head over to `data/global/config/channel-messenger.json`. If it doesn't exist, restart Botpress Server.
-- Set the following properties:
-  - `appSecret`. You will find this value in your Facebook App page.
-  - `verifyToken`. This is a random string you need to generate and keep secret. You'll need to copy/paste this token in the Facebook App portal when setting up your webhook.
-- Make sure you have an HTTPS url pointing to your Botpress Server and set the [`EXTERNAL_URL`](../advanced/configuration#exposing-your-bot-on-the-internet) environment variable
-- Restart Botpress Server to reload the configuration
-- Setup your webhook (see below)
+"`js
+"modules": [
+  ...
+    {
+      "location": "MODULES_ROOT/channel-messenger",
+      "enabled": true
+    },
+```
+
+### Get App Secret & Verify Token
+
+#### App Secret
+- Go to your Facebook App
+- In the left sidebar, expand the Settings menu and select Basic. Here you can find the App ID and  App Secret.
+- Click on the "Show" button in the "App Secret" text box. You can copy the "App Id" and "App Secret" to use for your Facebook API calls.
 
-#### Setup your webhook
+#### Verify Token
+You can set a long string of your own making as a verify token or use a random string generator service like random.org to create a verify token.
 
-Messenger will use a webhook that you'll need register in order to communicate with your bot.
+## Configurations
+The following configurations are required for the connection to be established to enable you to access your chatbot from your Facebook page. 
+
+While the first three are mandatory, the last three configurations (greeting text, get started, and persistent menu) are not.
 
-1. In Products > Messenger > Settings > Webhooks > Setup Webhooks
-1. In Callback URL, enter your secured public url and make sure to point to the `/webhook` endpoint.
-1. Paste your verifyToken (the random string that you generated) in the Verify Token field.
-1. Make sure `messages` and `messaging_postbacks` are checked in Subscription Fields.
 
-> **⭐ Note**: When you setup your webhook, Messenger requires a **secured public** address. To test on localhost, we recommend using services like [pagekite](https://pagekite.net/), [ngrok](https://ngrok.com) or [tunnelme](https://localtunnel.github.io/www/) to expose your server.
+### Botpress HTTPS Endpoint 
+- Head over to `data/global/config/channel-messenger.json`. If it doesn't exist, restart Botpress Server.
```

Comment:
I think we should rather instruct the user to modify the file through the Code Editor. I feel like most users will use the Code Editor to modify files, rather than head over to the `data/` directory on their local computer. This is even more true for deployments that use Postgresl as the file backend: they won't find the `data/` directory locally on their Botpress host server.

Reply:
Alright